src/vs/workbench/contrib/bracketPairColorizer2Telemetry/browser/bracketPairColorizer2Telemetry.contribution.ts

```diff
@@ -45,7 +45,7 @@ class BracketPairColorizer2TelemetryContribution {
 		type BracketPairColorizer2Event = {
 			nativeColorizationEnabled: boolean;
 		};
-		this.telemetryService.publicLog2<BracketPairColorizer2Event, BracketPairColorizer2InstalledClassification>('bracketPairColorizer2Usage', {
+		this.telemetryService.publicLog2<BracketPairColorizer2Event, BracketPairColorizer2InstalledClassification>('bracketPairColorizerTwoUsage', {
```

Comment:
@lramos15 Is there something we could do here or in tooling to catch cases like this sooner? I was not aware that digits are disallowed in event names.

Reply:
Yep already ahead of you https://github.com/microsoft/vscode-telemetry-extractor/pull/13 👍